.github/workflows/storybook.yml

```diff
@@ -147,3 +147,38 @@ jobs:
           git pull --no-edit --quiet
           git commit -m "Remove storybook for pull/${PULL_REQUEST_NUMBER}."
           git push origin gh-pages
+
+  test-storybook:
+    name: Test Storybook
+    runs-on: ubuntu-latest
+    # Tried this but keeps failing
+    # needs: build-storybook
```

Comment:
Yesterday this step kept failing when I used the `needs` step (although I'm sure it was working before :thinking:  )  @aaemnnosttv 

I can look into this more today, and see if there's anything problematic with this, or if I'm doing something wrong 

An alternative would be to combine the build and test steps (like we combined the various JS steps).  If the test fails then I suppose we don't want to do anything else with storybook and want to bail as quickly as possible!